src/App.css

```diff
@@ -1891,17 +1886,224 @@ border-radius: 8px;
   }
 }
 
+.add-button {
+  background-color: #007bff; /* Blue color */
+  color: white;
+  padding: 8px 16px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer; /* Center the button horizontally */
+  display: block; /* Ensure the button is a block element */
+  margin: 10px auto;
+}
+
+.add-button:hover {
+  background-color: #0056b3; /* Darker blue color on hover */
+}
+
+.add-button:disabled {
+  background-color: #ccc; /* Gray color for disabled button */
+  color: #666;
+  cursor: not-allowed;
+}
+
+.Modal {
```

Comment:
lines 1910-1932 seem to be adding new styling for these? I thought we changed it back to the old one. Regardless, can you check
1. If this is the styling that you have locally right now.
2. If the modal styling interferes with any of the modals - testing out the bios modals in the landing page, the create group modal, and the add members modal.